ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/metacard-history/container.tsx

```diff
@@ -54,7 +54,12 @@ class MetacardHistory extends React.Component<Props, State> {
   loadData() {
     setTimeout(async () => {
       const res = await fetch(
-        `./internal/history/${this.model.get('metacard').get('id')}`
+        `./internal/history/${this.model
+          .get('metacard')
+          .get('id')}?sourceId=${this.model
+          .get('metacard')
+          .get('properties')
+          .get('source-id')}`
```

Comment:
❗ This will not work with harvested data since the source id for harvested data is stored on a different attribute.

Reply:
Tested this while fixing a bug in another area. The `source-id` property did contain the harvested source's id. Not sure if accessing it this way will not work however it's definitely something that needs to be tested.